Realm/RLMSyncSessionRefreshHandle.mm

```diff
@@ -40,6 +40,8 @@ void unregisterRefreshHandle(const std::weak_ptr<SyncUser>& user, const std::str
 
 }
 
+static const NSTimeInterval RefreshBuffer = 10;
```

Comment:
Based on `RLMCollection.mm`, can you rename this `RLMRefreshBuffer` just so it's clear that it's a static constant?